nimhapkg/resources/utils/common.nim

```diff
@@ -0,0 +1,23 @@
+
+import parsecfg
+from os import getAppDir
+from strutils import replace
+
+proc loadConf*(modulename: string): Config =
+  ## Load config for the main daemon or a module
+  var fn = ""
+  when defined(dev):
+    if modulename == "":
+      # main daemon
+      fn = getAppDir() & "/config/secret.cfg"
+    else:
+      fn = replace(getAppDir(), "/nimhapkg/mainmodules", "") & "/config/secret.cfg"
+  else:
+    if modulename == "":
+      fn = "/etc/nimha/nimha.cfg"
```

Comment:
`/etc/nimha/nimha.cfg` does not exists. It needs to created.

If we are going to move the `config_default.cfg` to both `getAppDir() & "/config/secret.cfg"` and `/etc/nimha/nimha.cfg`, we should inform the user that he needs to update the config in 2 places. Or an implementation where the user could edit the config in the browser, where the backend updates 2 places; or a simple symlink.

https://github.com/ThomasTJdev/nim_homeassistant/blob/2c28eab8e1c152d20b64074f00ab437e2d58422c/nimha.nim#L38

Reply:
Configs under /etc must be treated as read-only by applications. If you want to edit them from the browser we can switch to /var/lib/nimha/config/

Reply:
/etc would be fine - the current options in the config file should be more or less static values. The different modules keeps their config in databases.

But I still think, that we should help or inform the user to move the `/config/secret.cfg` to `/etc/nimha/nimha.cfg`. Currently we generate the raw `secret.cfg` in `proc secretCfg()`, but that'll only work for `-d:dev`. We should support an plug-and-play solution.

Info: the name `secret.cfg` is derived from Home Assistant. We can change it to nimha.cfg if you like.